quilt/tools/build.py

```diff
@@ -13,6 +14,19 @@ class BuildException(Exception):
     """
     pass
 
+def _pythonize_name(name):
+    safename = re.sub('[^A-Za-z0-9_]+', '_', name)
+    starts_w_number = re.match('^[0-9].*', safename)
+    if starts_w_number:
+        safename = ("a%s" % safename)
+
+    safename = safename.rstrip('_')
+    safename = safename.lstrip('_')
+
+    if not VALID_NAME_RE.match(safename):
+        raise BuildException("Unable to determine a Python-legal name for %s" % name)
```

Comment:
Good suggestion, but this method isn't the right place for that logic since we don't know if it's a file, table or other.